deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: knative-eventing-operator
           # Replace this with the built image name
-          image: quay.io/matzew/knative-eventing-operator:v0.5.0
+          image: quay.io/openshift-knative/knative-eventing-operator:v0.5.0
```

Comment:
@jcrossley3 on "serving" it does still point to your org

https://github.com/openshift-knative/knative-serving-operator/blob/master/deploy/operator.yaml#L18